reV/supply_curve/exclusions.py

```diff
@@ -247,19 +321,45 @@ def _check_mask_type(self):
                         logger.error(msg)
                         raise ExclusionLayerError(msg)
 
-        if mask == 'inclusion_weights' and self._weight < 1:
+        if mask == 'include_weights' and self._weight < 1:
             msg = ("Values are individually weighted when using "
-                   "'inclusion_weights', the supplied weight of {} will be "
+                   "'include_weights', the supplied weight of {} will be "
                    "ignored!".format(self._weight))
             self._weight = 1
             logger.warning(msg)
             warn(msg)
 
         return mask
 
-    def _range_mask(self, data):
+    def _exclude_range_mask(self, data):
         """
-        Mask exclusion layer based on value range
+        Mask exclusion layer based on exclude value range
+
+        Parameters
+        ----------
+        data : ndarray
+            Exclusions data to create mask from
+
+        Returns
+        -------
+        mask : ndarray
+            Boolean mask of which values to include (True is include).
+        """
+        mask = True
```

Comment:
So I did not know that you could index into a `np.array` using just `True`, but I just tried it and it indeed works. However, it seems that doing so changes the shape of the array:

```
>>> import numpy as np
>>> a = np.arange(5)
>>> a
array([0, 1, 2, 3, 4])
>>> a.shape
(5,)
>>> a[True]
array([[0, 1, 2, 3, 4]])
>>> a[True].shape
(1, 5)
```

Do you think this would be an issue? Would it be worthwhile to add a test somewhere that has `exclude_range` set to something that does not intersect with the data? I'm not familiar enough with the rest of the exclusion code to know if this would make a difference

Reply:
oh weirdddd. I just changed the init value of the mask to an np.fill. that should fix it.

Reply:
Fair enough!